specs/schema.TEStribute.openapi.yaml

```diff
@@ -0,0 +1,96 @@
+openapi: "3.0.0"
+info:
+  version: 0.0.1
+  title: TEStribute OpenAPI specification
+  contact:
+    name: 'Elixir Europe'
+    email: 'alexander.kanitz@alumni.ethz.ch'
+  license:
+    name: Apache 2.0
+    url: 'https://github.com/elixir-europe/TEStribute/blob/dev/LICENSE'
+paths:
+  /rank_services:
+    post:
+      operationId: rank_services
+      description: Creates a new pet in the store.  Duplicates are allowed
+      requestBody:
+        description: Pet to add to the store
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/rank_services_requirements'
+      responses:
+        '200':
+          description: ranking successful
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/rank_services_response'
+
+components:
+  schemas:
+    rank_services_requirements:
+      type: object
+      properties:
+        drs_ids:
+          $ref: '#/components/schemas/drs_ids'
+        drs_uris:
+          $ref: '#/components/schemas/drs_uris'
+        mode:
+          anyOf:
+            - type: integer
```

Comment:
but should be enum of -1, 0, 1

Reply:
I think you've modified this to allow quite a few options in the `validate_inputs` module. I might have understood incorrectly I will check and modify accordingly.